apex_launchtest/apex_launchtest/apex_runner.py

```diff
@@ -27,14 +27,7 @@
 from .io_handler import ActiveIoHandler
 from .loader import PostShutdownTestLoader, PreShutdownTestLoader
 from .proc_info_handler import ActiveProcInfoHandler
-from .test_result import TestResult
-
-
-class _fail_result(unittest.TestResult):
```

Comment:
It was kind of a hack to put this in here.  I moved it to be next to the real TestResult so hopefully any new attributes that get added to one have a chance of being added to the other